plugins/org.polarsys.capella.scenario.editor.embeddededitor/src/org/polarsys/capella/scenario/editor/embeddededitor/commands/XtextToDiagramCommands.java

```diff
@@ -788,6 +832,19 @@ private static boolean foundCapellaMessageInXText(SequenceMessage capellaSequenc
     return false;
   }
 
+  
+  private static boolean foundCapellaStateFragmentInXText(Scenario scenario, TimeLapse timelapse,
```

Comment:
if you remove scenario from isSameStateFragment, you could remove it from this method too

Reply:
Done.